src/apiCalls.js

```diff
@@ -14,3 +14,8 @@ export const submitLoginCredentials = (state) => {
   })
   .then(response => response.json())
 }
+
+export const getMovieDetails = movieId => {
```

Comment:
We will also need a getMovieVideos function here because the videos are at a different url. This will then need to be added into the MovieDetails component!